bitnami/etcd/templates/statefulset.yaml

```diff
@@ -27,10 +27,10 @@ spec:
     metadata:
       name: "{{ template "etcd.fullname" . }}"
       labels:
-        app: {{ template "etcd.name" . }}
-        chart: {{ template "etcd.chart" . }}
-        release: {{ .Release.Name | quote }}
-        heritage: {{ .Release.Service | quote }}
+        app.kubernetes.io/name: {{ template "etcd.name" . }}
```

Comment:
Changing the templates in a StatefulSet would cause helm upgrade from a previous version fail, right?

Reply:
Yes, it's documented in the **README.md**
This PR breaks backwards compatibility